src/strategies/iotex-balance/index.ts

```diff
@@ -0,0 +1,52 @@
+import { strategy as EthBalanceStrategy } from '../eth-balance';
+import fetch from 'cross-fetch';
+
+interface ApiReturn {
+  balance: string;
+}
+
+export const author = 'iotex';
+export const version = '0.0.1';
+
+const testNetUrl = 'https://testnet.iotexscout.io/apiproxy';
+const mainNetUrl = 'https://iotexscout.io/apiproxy';
+
+function getUrl(network) {
+  return (network == 4689) ? mainNetUrl : testNetUrl;
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  if (blockTag == 'latest')
+    return EthBalanceStrategy(space, network, provider, addresses, options, snapshot);
+
+  const apiUrl = getUrl(network);
+  const promisesBalances = addresses.map(v => {
```

Comment:
This will not scale well, if there is 1000 voter it will send 1000 requests. Do you have another way to gather balance for multiple addresses with a single request?

Reply:
change the request, gather balances for multiple addresses with a single request.
please check it, thanks
@bonustrack @ChaituVR